lib/manifestly/manifest.rb

```diff
@@ -40,6 +40,9 @@ def self.read(filename, repositories)
 
     def write(filename)
       File.open(filename, 'w') do |file|
+        @items.sort_by! do |item|
+          item[:repository_name]
+        end
```

Comment:
fyi, should have been `item.repository_name` instead of `item[:repository_name]`